Infra/Indexers/usdc.indexer.js

```diff
@@ -0,0 +1,66 @@
+import { hash, uint256 } from "https://esm.run/starknet@5.14";
+import { formatUnits } from "https://esm.run/viem@1.4";
+
+const USDC_ADDRESS =
+  "0x53c91253bc9682c04929ca02ed00b3e423f6710d2ee7e0d5ebb06f3ecf368a8";
+
+const USDC_DECIMALS = 6;
+
+const usdcTransferFilter = {
+  header: {
+    weak: true,
+  },
+  events: [
+    {
+      fromAddress: USDC_ADDRESS,
+      keys: [hash.getSelectorFromName("Transfer")],
+      includeReceipt: false,
+    },
+  ],
+};
+
+function decodeUSDCTransfer({ header, events }) {
+  const { blockNumber, blockHash, timestamp } = header;
+  return (events ?? []).map(({ event, transaction }) => {
+    const transactionHash = transaction.meta.hash;
+    const transferId = `${transactionHash}_${event.index ?? 0}`;
+
+    const [from_address, to_address, amount_addedLow, amount_addedHigh] =
+      event.data;
+
+    const amount_bn = uint256.uint256ToBN({
+      low: amount_addedLow,
+      high: amount_addedHigh,
+    });
+    const amount = formatUnits(amount_bn, USDC_DECIMALS);
+
+    const block_timestamp = (new Date(timestamp).getTime() / 1000).toString();
+
+    return {
+      network: "starknet-mainnet",
+      block_hash: blockHash,
+      block_number: +blockNumber,
+      block_timestamp: block_timestamp,
```

Comment:
```suggestion
      block_timestamp: timestamp,
```

Then PostgreSQL will automatically convert the timestamp string to timestamp